sio/workers/test/test_executors.py

```diff
@@ -826,3 +834,16 @@ def test_execute():
         eq_(rc, 0)
         rc, out = execute(['ls', tempcwd()])
         in_(b'spam', out)
+
+
+def test_checker_percentage_parsing():
```

Comment:
Since some corner-cases are allowed, it might be useful to add them:
* "42."
* "007" & "0042"
* "" (empty string)
and invalid strings:
* "abc"
* "1e10"